charts/kuberpult/templates/cd-service.yaml

```
@@ -73,7 +72,46 @@ spec:
       imagePullSecrets:
         {{- toYaml . | nindent 8 }}
       {{- end }}
+      serviceAccountName: {{ .Values.cd.db.k8sName }}
```

this needs an `if`
```suggestion
 {{ if k8sname... }}
      serviceAccountName: {{ .Values.cd.db.k8sName }}
 {{ end }}
```
(pseudo code!)